.expeditor/scripts/finish_release/cargo_update.sh

```diff
@@ -18,8 +18,8 @@ install_hub
 echo "--- :habicat: Installing and configuring build dependencies"
 hab pkg install core/zeromq
 
-PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(< "$(hab pkg path core/zeromq)"/PKG_CONFIG_PATH)"
-export PKG_CONFIG_PATH 
+PKG_CONFIG_PATH="$(hab pkg path core/zeromq)/PKG_CONFIG_PATH"
```

Comment:
I  think you wan to keep the  `$(< `, otherwise the path will be set to the path of that file, rather than the contents.

Reply:
:man_facepalming: